src/main/java/seedu/taassist/logic/commands/AddcCommand.java

```diff
@@ -32,7 +32,9 @@ public class AddcCommand extends Command {
     private final Set<ModuleClass> moduleClasses;
 
     /**
-     * Creates an AddCommand to add the specified {@code Set&gt;ModuleClass&lt;}.
+     * Creates an AddCommand to add the specified {@code moduleClasses}.
```

Comment:
```suggestion
     * Creates an AddCommand to add the specified {@code moduleClasses}-es.
```
Perhaps this would be clearer of the method's intentions?

Reply:
`{@code moduleClasses}` is in plural form so I think it should be fine